docs/synapse/kubernetes/sysctl.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: "apps/v1"
+kind: "DaemonSet"
+metadata:
+  name: "setsysctl"
+  labels:
+    app.kubernetes.io/name: "sysctl"
+    app.kubernetes.io/instance: "sysctl"
+    app.kubernetes.io/version: "1.36.0-glibc"
+    app.kubernetes.io/component: "sysctl"
+    app.kubernetes.io/part-of: "synapse"
+    environment: "dev"
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: "sysctl"
+      app.kubernetes.io/instance: "sysctl"
+      app.kubernetes.io/version: "1.36.0-glibc"
+      app.kubernetes.io/component: "sysctl"
+      app.kubernetes.io/part-of: "synapse"
+      environment: "dev"
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: "sysctl"
+        app.kubernetes.io/instance: "sysctl"
+        app.kubernetes.io/version: "1.36.0-glibc"
+        app.kubernetes.io/component: "sysctl"
+        app.kubernetes.io/part-of: "synapse"
+        environment: "dev"
+    spec:
+      containers:
+      - name: "apply-sysctl"
+        image: "busybox:1.36.0-glibc"  # Latest glibc based busybox
+        securityContext:
+          privileged: true
+        command:
+        - "/bin/sh"
+        - "-c"
+        - |
+          set -o errexit
+          set -o xtrace
+          while sysctl -w vm.swappiness=10 vm.dirty_expire_centisecs=20 vm.dirty_writeback_centisecs=20 vm.dirty_background_ratio=2 vm.dirty_ratio=4
```

Comment:
Does the deployment include this image side-saddled with all the others? Do we need to specify one of these runs per "actual host" or anything?

Reply:
That is a function of how a DaemonSet in kubernetes - to run on all (specified ) nodes. This DaemonSet does run on all nodes in a cluster. An operator that needs to deploy something like this to a specific subset of nodes would use normal nodeSelector / affinity mechanisms.

We can definitely highlight that the example runs on all nodes if I did not call that out alreayd.